support/templates/partials/treatment.hbs

```diff
@@ -56,26 +56,37 @@
-        </p>
-        <p>
-          Totaal aantal onthoudingen, blanco of ongeldig:
-          <span datatype="xsd:integer" property="besluit:aantalOnthouders" content="{{this.abstentionVotes}}">{{this.abstentionVotes}}</span>
-        </p>
-        {{#unless this.geheim}}
-          {{> mandateeList mandatees=this.positiveVoters property="besluit:heeftVoorstander" title="Voorstanders" includeRole=false}}
-          {{> mandateeList mandatees=this.negativeVoters property="besluit:heeftTegenstander" title="Tegenstanders" includeRole=false}}
-          {{> mandateeList mandatees=this.abstentionVoters property="besluit:heeftOnthouders" title="Onthoudingen, blanco of ongeldig" includeRole=false}}
-        {{/unless}}
-        <p>
-          Met als gevolg,
-        </p>
+        {{#if this.isUnanimous}}
+          <span datatype="xsd:integer" property="besluit:aantalVoorstanders" content="{{this.positiveVotes}}"></span>
+          <span datatype="xsd:integer" property="besluit:aantalTegenstanders" content="{{this.negativeVotes}}"></span>
+          <span datatype="xsd:integer" property="besluit:aantalOnthouders" content="{{this.abstentionVotes}}"></span>
+          {{#unless this.geheim}}
+            {{> mandateeList mandatees=this.positiveVoters property="besluit:heeftVoorstander" title="Voorstanders" includeRole=false hideText=true}}
+            {{> mandateeList mandatees=this.negativeVoters property="besluit:heeftTegenstander" title="Tegenstanders" includeRole=false hideText=true}}
+            {{> mandateeList mandatees=this.abstentionVoters property="besluit:heeftOnthouders" title="Onthoudingen, blanco of ongeldig" includeRole=false hideText=true}}
+          {{/unless}}
+          <p>Met eenparigheid van stemmen,</p>
+        {{else}}
+          <p>
+            Totaal aantal voorstanders:
+            <span datatype="xsd:integer" property="besluit:aantalVoorstanders" content="{{this.positiveVotes}}">{{this.positiveVotes}}</span>
+          </p>
+          <p>
+            Totaal aantal tegenstanders:
+            <span datatype="xsd:integer" property="besluit:aantalTegenstanders" content="{{this.negativeVotes}}">{{this.negativeVotes}}</span>
+          </p>
+          <p>
+            Totaal aantal onthoudingen, blanco of ongeldig:
+            <span datatype="xsd:integer" property="besluit:aantalOnthouders" content="{{this.abstentionVotes}}">{{this.abstentionVotes}}</span>
+          </p>
+          {{#unless this.geheim}}
+            {{> mandateeList mandatees=this.positiveVoters property="besluit:heeftVoorstander" title="Voorstanders" includeRole=false}}
+            {{> mandateeList mandatees=this.negativeVoters property="besluit:heeftTegenstander" title="Tegenstanders" includeRole=false}}
+            {{> mandateeList mandatees=this.abstentionVoters property="besluit:heeftOnthouders" title="Onthoudingen, blanco of ongeldig" includeRole=false}}
+          {{/unless}}
+          <p>Met als gevolg,</p>
```

Comment:
this should be outside of the if block, it should also show when its unanimous

Reply:
I thought it was replaced by the `Met eenparigheid van stemmen` but I will put it outside :)